playbooks/inventory_workflow_manager.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Configure device credentials on Cisco DNA Center
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+    - "input_inventory.yml"
+    - "credentials.yml"
+  tasks:
+    - name: Add/Update/Resync/Delete the devices in Cisco DNA Center Inventory.
+      cisco.dnac.inventory_workflow_manager:
+        dnac_host: "{{dnac_host}}"
+        dnac_username: "{{dnac_username}}"
+        dnac_password: "{{dnac_password}}"
+        dnac_verify: "{{dnac_verify}}"
+        dnac_port: "{{dnac_port}}"
+        dnac_version: "{{dnac_version}}"
+        dnac_debug: "{{dnac_debug}}"
+        dnac_log_level: DEBUG
+        dnac_log: true
+        config_verify: true
+        state: merged
+        config:
+        - username: "{{item.username}}"
+          enable_password: "{{item.enable_password}}"
+          password: "{{item.password}}"
+          ip_address: "{{item.ip_address}}"
+          cli_transport: "{{item.cli_transport}}"
+          # hostname_list: "{{item.hostname_list}}"
```

Comment:
We can remove the commented items or can use it

Reply:
we can remove it.